templates/prescription/general_edit.html

```diff
@@ -295,6 +295,13 @@
 }
 
 $().ready(function() {
+
+    $("#save,#back").on("click",function(){
+        $("#clearButton",window.parent.document).css("display", "none");
+        $("#backButton",window.parent.document).css("display", "none");
+        $("#addButton",window.parent.document).css("display", "inline-block");
```

Comment:
@eyalvo-matrix 
This is an example of if we wanted to change the original buttons defined in dialog to block type block we'd have to remember to change here as well. Where if display is set to display:'' then the block type would remain the same as the original style ie inherited.
Also I think you still need to add the context button change
`
$("#clearButton",window.parent.document).css("display", "");
$("#backButton",window.parent.document).css("display", "");
$("#addButton",window.parent.document).css("display", "none");
`
Where entire block is:
`
$("#save,#back").on("click",function() {
        $("#clearButton", window.parent.document).css("display", "none");
        $("#backButton", window.parent.document).css("display", "none");
        $("#addButton", window.parent.document).css("display", "");
    });
$("#clearButton",window.parent.document).css("display", "");
$("#backButton",window.parent.document).css("display", "");
$("#addButton",window.parent.document).css("display", "none");
`

Reply:
@sjpadgett thank you for your comment, I guess it is better I also added the context as you suggested though it did work without it but maybe there is a scenario that does not come to my mind :)

Reply:
Without them then when in edit mode the dialog buttons do not change. The iframes back button works great though. Tested on FF Quantum and Edge.